ST_environs.c

```diff
@@ -102,51 +89,36 @@ static void _make_ppt( void) {
 
 /*------------------------------------------------------*/
 
-  IntS r=0, i;
+  IntS i;
 
 #ifdef DEBUG_ENVCONST
-  r=320;
+  IntS r=320;
 #endif
 
-  if (UseSoilwat)
-  { // Run with SOILWAT2: we have monthly PPT and temperature to calculate
-    // growing season precipitation as sum of monthly precipitation of those
-    // months when mean air temperature exceeds a threshold `GROWING_BASE_TEMP`
-    Env.gsppt = 0; // gsppt is defined as IntS and units are millimeters
-
-    for (i = 0; i < MAX_MONTHS; i++)
-    {
-      Env.gsppt += GE(SXW.temp_monthly[i], GROWING_BASE_TEMP) ?
-        (IntS) (SXW.ppt_monthly[i] * 10. + 0.5) : 0;
-    }
-
-    if (Env.gsppt <= 0)
-    {
-      LogError(logfp, LOGWARN, "Zero growing season precipitation in "\
-        "year = %d of iteration = %d", Globals.currYear, Globals.currIter);
-      Env.gsppt = 0;
-    }
+  // Run with SOILWAT2: we have monthly PPT and temperature to calculate
+  // growing season precipitation as sum of monthly precipitation of those
+  // months when mean air temperature exceeds a threshold `GROWING_BASE_TEMP`
+  Env.gsppt = 0; // gsppt is defined as IntS and units are millimeters
 
-    /*
-    printf("_make_ppt (year = %d of iteration = %d): "\
-      "Env.gsppt(SOILWAT2) = %d would be gsppt(fixed proportion) = %.1f\n",
-      Globals.currYear, Globals.currIter, Env.gsppt, Globals.gsppt_prop * Env.ppt);
-    */
-
-  } else {
-    // run as STEPPE without SOILWAT2:
-    while ( r < Globals.ppt.min || r > Globals.ppt.max )
-      r = (IntS)(RandNorm( Globals.ppt.avg,  Globals.ppt.std, &environs_rng) +.5);
-    if (Env.ppt > 0) {
-      Env.lyppt = Env.ppt;
-      Env.ppt = r;
-    } else {
-      Env.lyppt = Env.ppt = r;
-    }
+  for (i = 0; i < MAX_MONTHS; i++)
+  {
+    Env.gsppt += GE(SXW.temp_monthly[i], GROWING_BASE_TEMP) ?
+      (IntS) (SXW.ppt_monthly[i] * 10. + 0.5) : 0;
+  }
 
-    Env.gsppt = (IntS) (Globals.gsppt_prop * Env.ppt);
+  if (Env.gsppt <= 0)
+  {
+    LogError(logfp, LOGWARN, "Zero growing season precipitation in "\
+      "year = %d of iteration = %d", Globals.currYear, Globals.currIter);
+    Env.gsppt = 0;
   }
 
+  /*
+  printf("_make_ppt (year = %d of iteration = %d): "\
+    "Env.gsppt(SOILWAT2) = %d would be gsppt(fixed proportion) = %.1f\n",
+    Globals.currYear, Globals.currIter, Env.gsppt, Globals.gsppt_prop * Env.ppt);
+  */
+
```

Comment:
any reason to keep this printf statement?

Reply:
Commit 51d6f29 removes this.